text_lloom/src/text_lloom/llm.py

```diff
@@ -63,6 +63,12 @@
     "text-embedding-3-large": (0.00013/1000),
 }
 
+EMBED_MAX_LENGTHS = {
+    "text-embedding-ada-002": 2000,
+    "text-embedding-3-small": 2000,
+    "text-embedding-3-large": 2000,
+}
+
```

Comment:
Thanks so much for working on this to support longer doc sets! Minor, but could we call this something like `EMBED_MAX_BATCH_SIZE` to clarify that it's the number of documents (as opposed to the number of tokens). Also, it looks like the max batch size might be 2048 (based on [this thread](https://community.openai.com/t/embeddings-api-max-batch-size/655329/2), though I couldn't find it in the documentation), so perhaps we can list 2048 here if that's correct.

Reply:
I just went ahead and made those changes since they were very minor! Thanks again for making the PR.